project.clj

```diff
@@ -6,11 +6,12 @@
   :source-paths ["src/clojure"]
   :java-source-paths ["src/java"]
   :profiles {:dev {:dependencies [[jline "0.9.94"]
-                                  [org.apache.directory.server/apacheds-all "1.5.5"]
+                                  [org.apache.directory.server/apacheds-all "1.5.7"
+                                   ;; This dependency causes the classpath to contain two copies of the schema,
+                                   ;; which prevents the test DS from starting
```

Comment:
(nit) I prefer spelling things out to using acronyms, I imagine not everyone would know what this meant at a glance.
```suggestion
                                   ;; which prevents the test Directory Service from starting
```